tuhi/dbusserver.py

```diff
@@ -391,6 +396,7 @@ def _start_live(self, connection, sender, args):
         logger.debug(f'Live mode started on {self.name} for {sender}')
 
         self.live = True
+        self._uhid = args[0]
```

Comment:
In regards to this not working, did you print the type of `args[0]` here? maybe it's still wrapped in something?

Reply:
could be, I'll have a look